### ISSUE TITLE
ovalocytes

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4874]
**Preferred term label:**

Ovalocytes

**Synonyms**

Elliptocytes 

**Definition (free text, please give PubMed ID)**

Ovalocytes are abnormal erythrocytes ranging from slightly oval to elongated cigar-shaped forms.

PMID: 21250106

**Parent term (use hpo.jax.org/app)**
http://purl.obolibrary.org/obo/HP_0001877

Abnormal erythrocyte morphology HP:0001877

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 166900
OVALOCYTOSIS

**Your nano-attribution (ORCID)**

0000-0001-9969-8610

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0004445
Adding Ovalocytes as a synonym.
